RELEASENOTES.md

```diff
@@ -20,14 +20,18 @@
 -->
 # Release Notes
 
+
+### 4.0.4 (Oct 02, 2017)
+* Display app version on splashscreen (Android, iOS)
```

Comment:
For the future, please don't edit release notes.  Since this is adding a new API, we would have to increment the minor release number instead of Trivial, and we have no idea when the exact release date is until we actually finish voting on a release, as per Apache policy.